<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FillPointDlg</class>
 <widget class="QDialog" name="FillPointDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>456</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>FillPointDlg</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="fillPointGroupBox">
     <property name="title">
      <string>填涂点定位</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QFormLayout" name="topLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>填涂点类型：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="typeCmb">
          <item>
           <property name="text">
            <string>选择题</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>座位号</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>考场号</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>试室号</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>首考生号</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>准考证号</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>科目号</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>缺考</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>试卷类型</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>选做题</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>违纪</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>页码</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>识别参考点</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="groupLabel">
          <property name="text">
           <string>标记点组号：</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="groupCmb"/>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="topLayout_2">
        <item>
         <widget class="QLabel" name="paibuLabel">
          <property name="text">
           <string>选项排布：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="horRadioBtn">
          <property name="text">
           <string>横向</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup_2</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="verRadioBtn">
          <property name="text">
           <string>纵向</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup_2</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="typeLabel">
          <property name="text">
           <string>题目类型：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="singleRadioBtn">
          <property name="text">
           <string>单选</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="multiRadioBtn">
          <property name="text">
           <string>多选</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="questionTypeCbx">
          <property name="text">
           <string>含ABCD</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="nameLabel">
            <property name="text">
             <string>名    称：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="nameEdit">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>以0开头</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="5" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="optionNumLabel">
          <property name="text">
           <string>题    号：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="optionEdit">
          <property name="toolTip">
           <string>可输入题号范围,如: 1-5</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="countLabel">
          <property name="text">
           <string>选项数量：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="numEdit">
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="hSpaceLabel">
          <property name="text">
           <string>横向间隔：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="rSpaceEdit">
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="vSpaceLabel">
          <property name="text">
           <string>纵向间隔：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="cSpaceEdit">
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="7" column="0">
       <layout class="QHBoxLayout" name="midLayout">
        <item>
         <widget class="QPushButton" name="addBtn">
          <property name="text">
           <string>添加填涂点</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="confirmBtn">
          <property name="text">
           <string>确认填涂区域</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="8" column="0">
       <layout class="QGridLayout" name="bottomLayout">
        <item row="0" column="1">
         <widget class="QPushButton" name="deleteBtn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>删除</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QListWidget" name="list">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
